### PR TITLE
Fix bleed being applied one turn in arena.

### PR DESCRIPTION
**Pull request by Unengine**
SSIA

### CODE REVIEW

File: .Lib9c.Tests/Action/BattleGrandFinale2Test.cs [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+    using Nekoyume.Model.GrandFinale;
+    using Nekoyume.Model.State;
+    using Nekoyume.TableData;
+    using Nekoyume.TableData.GrandFinale;
+    using Serilog;
+    using Xunit;
+    using Xunit.Abstractions;
+    using static Lib9c.SerializeKeys;
+
+    public class BattleGrandFinale2Test
```
Comment (boscohyun) [inline .Lib9c.Tests/Action/BattleGrandFinale2Test.cs:26]: How about this?

```suggestion
    public class BattleGrandFinaleTest
```

File: Lib9c/Action/BattleArena.cs [hunk 2]
```diff
@@ -24,6 +24,7 @@ namespace Nekoyume.Action
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/1663
     /// Hard forked at https://github.com/planetarium/lib9c/pull/1649
+    /// Updated at https://github.com/planetarium/lib9c/pull/1679
```
Comment (boscohyun) [inline Lib9c/Action/BattleArena.cs:27]: Does this `battle_arena8` not require to hard fork?
I guess the `ArenaSimulator` need to change to  `ArenaSimulatorV2` like the `battle_arena7`.

File: Lib9c/Model/Character/ArenaCharacter.cs [hunk 3]
```diff
@@ -300,6 +296,73 @@ private static CharacterSheet.Row CharacterRow(int characterId, ArenaSimulatorSh
             }
         }
 
+        public void SetRuneV2(
```
Comment (boscohyun) [inline Lib9c/Model/Character/ArenaCharacter.cs:299]: How about this?
- Add `Obsolete` attribute to `SetRune()`
- Rename `SetRune()` to `SetRuneV1()`
- Rename `SetRuneV2()` to `SetRune()`

File: Lib9c/Action/BattleArena.cs (hunk 2 above)
Comment (Unengine) [reply Lib9c/Action/BattleArena.cs:27]: `BattleArena8` is not released on mainnet yet.

**Review by hskim881028 — APPROVED** [review]

**Review by boscohyun — APPROVED** [review]

**Pull request merged**